packages/react-server/src/ReactFizzServer.js

```diff
@@ -334,6 +334,7 @@ export opaque type Request = {
   // rendering - e.g. to the client. This is considered intentional and not an error.
   onPostpone: (reason: string, postponeInfo: ThrownInfo) => void,
   // Form state that was the result of an MPA submission, if it was provided.
+  // TODO: is this actionState?
```

Comment:
@acdlite should this rename to `actionState`?

Reply:
This is specific to forms. It's the progressive enhancement feature.

Reply:
I thought that might be the case, thank for confirming. I'll remove the TODOs and give it another pass.